docs/Current_Code.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+import numpy
+from pyfirmata import Arduino, util
+import time
+from time import sleep
+from pyfirmata import INPUT, OUTPUT, PWM
+import serial
+import re
+import matplotlib.pyplot as plt
+
+class myArduinoBoard(object):
+    def __init__(self,portname = '/dev/tty.usbmodem14201'):
+        self.port = portname
+
+    def setup_arduino(self):
+        self.board = Arduino(self.port)
+        self.arduino = serial.Serial(self.port, 9600)
+        sleep(0.02)  # The synchronization between the Arduino board and pyFirmata requires some time. 1 = 1 second
+        self.analog_d9p = self.board.get_pin('d:9:p')  # Same as board.digital[9].mode = PWM
+        self.board.analog[0].mode = INPUT  # voltage
+        self.board.analog[2].mode = INPUT  # current
+        self.analog_d9p.frequency = 31372.25
+        self.it = util.Iterator(self.board)
+        self.it.start()
+        self.board.analog[0].enable_reporting()
+        self.board.analog[2].enable_reporting()
+
+    def readPin(self):
+        words = []
+        nums = []
+        while True:
+            line = self.arduino.readline()
+            if len(line) != 0:
+                words = re.findall(r'-?\d+.\d\d\d\d', line.decode('utf-8'))
+                nums = [float(w) for w in words]
+                print(nums)
+                if (len(nums) == 2):
+                    return nums
+
+    def readData(self,x):
```

Comment:
Maybe a better name is needed?  I am not sure what this method is doing but it seems that it is doing more than reading a quantity.